js/base/Exchange.js

```diff
@@ -2636,4 +2636,22 @@ module.exports = class Exchange {
             throw new NotSupported (this.id + ' fetchPremiumIndexOHLCV () is not supported yet');
         }
     }
+
+    timeInForce (params = {}) {
+        /**
+         * @ignore
+         * @method
+         * * Must add timeInForce to this.options to use this method
+         * @return {str} returns the exchange specific value for timeInForce
+         */
+        const timeInForce = this.safeStringUpper (params, 'timeInForce'); // supported values GTC, IOC, PO
+        if (timeInForce !== undefined) {
+            const exchangeValue = this.safeString (this.options['timeInForce'], timeInForce);
+            if (exchangeValue === undefined) {
+                throw new ExchangeError (this.id + ' does not support timeInForce "' + timeInForce + '"');
+            }
+            return exchangeValue;
+        }
+        return undefined;
+    }
```

Comment:
@kroitor Let me know what you think of this method, I can remove it right away if you don't like it, cause this bug should get fixed right away